### ISSUE TITLE
`odo dev` does not allow to run multiple parallel Dev sessions on different clusters

### ISSUE THREAD

**Issue by valaparthvi** [redhat-developer/odo#6809]
/kind user-story

## User Story
As an odo user, I want to be able to run 2 parallel Dev session on different clusters, So that I can simultaneously check my application/cluster's behavior instead of having to restart the Dev sessions and moving back and forth between the clusters.

## Acceptance Criteria 
- [ ] It should allow to run multiple dev sessions on different clusters.

Note: It might make sense to be able to do that on podman as well if it is possible to run multiple podman applications.

## Links 
- Related Epic (mandatory): 

/kind user-story

**Comment by feloy** [comment]
We could consider adding a DOT_ODO_DIRECTORY environment variable to change the directory on which devstate files are stored.

**Comment by valaparthvi** [comment]
@rm3l is this still relevant? I see it's pointed, so it was relevant at some point.

**Comment by rm3l** [comment]
> is this still relevant? I see it's pointed, so it was relevant at some point.

Makes sense to keep it open, indeed, as it had been pointed and prioritized.
Thanks for commenting out.

/remove-lifecycle stale
/lifecycle frozen

**Comment by rm3l** [comment]
Changing the title to make it clear that this is a new feature, not a bug.

/retitle Allow running multiple parallel Dev sessions on the same platform type
/remove-lifecycle frozen
